editoast/openapi.yaml

```diff
@@ -42,6 +42,62 @@ paths:
               schema: 
                 type: array
```

Comment:
I know it's not part of this PR, but that's not valid OpenAPI. Please use https://editor.swagger.io/ to validate your changes.

Reply:
Done in https://github.com/DGEXSolutions/osrd/pull/2917